app/components/CategoryOverviewView.js

```diff
@@ -0,0 +1,21 @@
+import React, { Children, PropTypes, cloneElement } from 'react';
+import { View } from 'react-native';
+
+/*
+ *Passing category to all children as prop
+ */
+const CategoryOverviewView = ({ category, children }) => (
+  <View>
+    {Children.map(children, child => cloneElement(child, { category }))}
```

Comment:
can you explain me why we need to do it this way and not with simple `<CategoryHeader {category=category}/>`?